src/web/interface/ConnectionBase.hpp

```diff
@@ -84,7 +84,7 @@ struct ConnectionBase : public util::Taggable {
      * @throws Not supported unless implemented in child classes. Will always throw std::logic_error.
      */
     virtual void
-    send(std::shared_ptr<std::string> /* msg */)
+    send([[maybe_unused]] std::shared_ptr<std::string> msg)
```

Comment:
@kuznetsss this is needed because `doxygen` complains `msg` is not a known param while generating the doc. Let's use `[[maybe_unused]]` as a convention from now on 👍